Missionframework/modules/11_permission/fnc/fn_permission_setupPlayerActions.sqf

```diff
@@ -19,7 +19,7 @@
 
 // Actions avalible LOCALLY to player
 // Permission action
-private _permissionCondition = '_target isEqualTo _originalTarget && serverCommandAvailable "#kick"';
+private _permissionCondition = '_target isEqualTo _originalTarget && (serverCommandAvailable "#kick" || (KPLIB_param_permissionCommander && (str player) isEqualTo "KPLIB_eden_commander") || KPLIB_param_permissionSubCommander && (str player) isEqualTo "KPLIB_eden_subCommander")';
```

Comment:
```suggestion
private _permissionCondition = '
    _target isEqualTo _originalTarget &&
    (serverCommandAvailable "#kick" ||
        (KPLIB_param_permissionCommander && (str player) isEqualTo "KPLIB_eden_commander") ||
        (KPLIB_param_permissionSubCommander && (str player) isEqualTo "KPLIB_eden_subCommander")
)';
```
Just a suggestion for a formatting which is easier to read for all valid cases (admin, commander or sub commander)

Reply:
The last bracket looks a little bit weird maybe we should bring it in a line with the bracket before `serverCommandAvailable` and push the command to the next line

Reply:
Like you prefer it. Not that important in the end.